### PR TITLE
mds: fix client capabilities during reconnect (client.XXXX isn't responding to mclientcaps(revoke))

### PR DESCRIPTION
**Pull request by ukernel**

### CODE REVIEW

**Comment by ukernel** [comment]
http://pulpito.ceph.com/ubuntu-2015-11-16_23:21:18-kcephfs-wip-11482-testing-basic-multi/
http://pulpito.ceph.com/ubuntu-2015-11-16_23:19:25-kcephfs-wip-11482-testing-basic-multi/

run some tests that the cap revoke warnings used to show

File: src/mds/MDCache.h [hunk 1]
```diff
@@ -491,7 +491,8 @@ class MDCache {
   map<inodeno_t,mds_rank_t> cap_export_targets; // ino -> auth mds
 
   map<inodeno_t,map<client_t,map<mds_rank_t,ceph_mds_cap_reconnect> > > cap_imports;  // ino -> client -> frommds -> capex
-  map<inodeno_t,filepath> cap_import_paths;
+  // map<inodeno_t,filepath> cap_import_paths;
```
Comment (jcsp) [inline src/mds/MDCache.h]: Commented out code should be removed

File: src/mds/MDCache.h [hunk 2]
```diff
@@ -534,7 +535,7 @@ class MDCache {
   void rejoin_recovered_caps(inodeno_t ino, client_t client, cap_reconnect_t& icr, 
 			     mds_rank_t frommds=MDS_RANK_NONE) {
     cap_imports[ino][client][frommds] = icr.capinfo;
-    cap_import_paths[ino] = filepath(icr.path, (uint64_t)icr.capinfo.pathbase);
+    // cap_import_paths[ino] = filepath(icr.path, (uint64_t)icr.capinfo.pathbase);
```
Comment (jcsp) [inline src/mds/MDCache.h]: commented out code again

**Comment by jcsp** [comment]
Let's merge this once the commented out lines are removed

**Comment by gregsfortytwo** [comment]
@ukernel, please describe the testing you performed on this PR.

**Comment by ukernel** [comment]
I used gdb to crash MDS when client flushes and release some caps at the same time. trigger mds failover and check mds log to if it works as expected

**Comment by gregsfortytwo** [comment]
http://pulpito.ceph.com/ubuntu-2015-11-24_12:59:17-fs-greg-fs-testing---basic-multi/ has more failures than I'd like but they're well-attributed. Please fix the commented code as John suggested and we'll merge this @ukernel. :)

**Comment by ukernel** [comment]
the commented code has already been removed

**Pull request merged**